fe/lib/components/BrandSelect/BrandSelect.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import 'braid-design-system/reset';
+
+import React, { ComponentProps } from 'react';
+
+import {
+  BraidArgs,
+  defaultArgTypes,
+  defaultArgs,
+} from '../../storybook/controls';
+import { BraidStorybookProvider } from '../../storybook/decorators';
+
+import { BrandSelect as Component } from './BrandSelect';
+import { MockBrandSelect } from './BrandSelect.mock';
+
+export default {
+  args: {
+    braidThemeName: defaultArgs.braidThemeName,
+    hirerId: 'seekAnz:organization:seek:kLDHs7W7',
+    initialBrandId:
+      'global:advertisementBranding:hirerBranding:4pkLmqYhoSxSKmfcKMbDG6',
+    pageSize: 4,
+  },
+  argTypes: {
+    braidThemeName: defaultArgTypes.braidThemeName,
+    showStorybookAction: defaultArgTypes.showStorybookAction,
+    pageSize: {
+      control: { type: 'radio' },
+      options: [4, 6],
+    },
```

Comment:
Do we need to explicitly mark this up, or can Storybook infer a numeric control?

```suggestion
```

Reply:
I got what you meant, the reason I only give it 2 options in pageSize that the mocked resolver handles for with max 3 pages on the same mocked dataset of 11 brands. If we allow to types pageSize, it doesn't know to handle dynamically in pages navigation. 

Any suggestion in the mock resolver more dynamically?

Reply:
Ah no stress. Feel free to merge as is, I'll take a look later.